packages/recurring-payments-job/src/recurring-payments-processor.js

```diff
@@ -107,11 +106,19 @@ const processRecurringPaymentStatus = async record => {
     state: { status }
   } = await getPaymentStatus(paymentId)
   debug(`Payment status for ${paymentId}: ${status}`)
-  if (status === PAYMENT_STATUS_SUCCESS) {
-    const payment = payments.find(p => p.paymentId === paymentId)
+  const payment = payments.find(p => p.paymentId === paymentId)
+  if (status === PAYMENT_STATUS.Success) {
     await salesApi.processRPResult(payment.transaction.id, paymentId, payment.created_date)
     debug(`Processed Recurring Payment for ${payment.transaction.id}`)
   }
+  if (status === PAYMENT_STATUS.Failure || status === PAYMENT_STATUS.Error) {
+    console.log(`Payment failed. Recurring payment agreement for: ${agreementId} set to be cancelled. Updating payment journal.`)
```

Comment:
I'm not sure this will trigger an error that'll be logged out to errbit and appear in our slack channel - I think you might need to use `console.error`. Please check this - you could deploy to dev and force an error to do so.